File: app/src/main/java/ch/unstable/ost/views/lists/station/SectionStationViewHolder.java
package ch.unstable.ost.views.lists.station;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.TextView;

import ch.unstable.ost.R;
import ch.unstable.ost.views.StopDotView;

public class SectionStationViewHolder extends RecyclerView.ViewHolder {

    public final TextView stationName;
    public final TextView stationTime;
    public final StopDotView stopDotView;

    public SectionStationViewHolder(View itemView) {
        super(itemView);
        stationName = itemView.findViewById(R.id.stationName);
        stationTime = itemView.findViewById(R.id.stationTime);
        stopDotView = itemView.findViewById(R.id.stopDotView);
    }
}

File: app/src/main/java/ch/unstable/ost/views/lists/station/SectionStationViewHolder.kt
package ch.unstable.ost.views.lists.station

import android.view.View
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView

import ch.unstable.ost.R
import ch.unstable.ost.views.StopDotView

class SectionStationViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    /**
     * The name of the passing station
     */
    val stationName: TextView = itemView.findViewById(R.id.stationName)
    /**
     * The time of the stop (arrival and departure)
     */
    val stationTime: TextView = itemView.findViewById(R.id.stationTime)
    /**
     * The stop-dot-view
     */
    val stopDotView: StopDotView = itemView.findViewById(R.id.stopDotView)

}
